src/services/pull/usePullCompareTotalsTeam.tsx

```diff
@@ -23,14 +24,16 @@ const CoverageObjSchema = z.object({
 const ImpactedFilesSchema = z.discriminatedUnion('__typename', [
   z.object({
     __typename: z.literal('ImpactedFiles'),
-    results: z.array(
-      z.object({
-        headName: z.string().nullable(),
-        missesCount: z.number(),
-        isCriticalFile: z.boolean(),
-        patchCoverage: CoverageObjSchema,
-      })
-    ),
+    results: z
+      .array(
+        z.object({
+          headName: z.string().nullable(),
+          missesCount: z.number(),
+          isCriticalFile: z.boolean(),
+          patchCoverage: CoverageObjSchema,
+        })
+      )
+      .nullable(),
```

Comment:
Ah, I see that `ImpactedFiles` object is nullable and that was missing across all the usages. It looks like it is marked as nullable on the `api` side as well so there may have been drift across the repos